src/bp/ui-studio/src/web/components/Layout/index.tsx

```diff
@@ -78,6 +85,7 @@ const Layout: FC<ILayoutProps> = props => {
   }, [props.translations])
 
   const toggleEmulator = () => {
+    setEmulatorOpen(!emulatorOpen)
     window.botpressWebChat.sendEvent({ type: 'toggle' })
     document.getElementById('main-content-wrapper').classList.toggle('emulator-open')
   }
```

Comment:
This work if you toggle using the button, but it doesn't if you toggle it using "e". Please take a look at `handleWebChatPanel` which I added some time ago to fix a similar issue with the inspector not following the open/close status of the webchat

Reply:
:+1: